packages/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -0,0 +1,745 @@
+<template>
+  <KPop
+    :hide-popover="hidePopover"
+    class="k-datetime-picker"
+    placement="bottomStart"
+    width="auto"
+    hide-caret
+    position-fixed
+    @closed="handleClose"
+  >
+    <KButton
+      :is-rounded="false"
+      size="large"
+      class="timepicker-input"
+      data-testid="k-datetimepicker-display"
+    >
+      {{ abbreviatedDisplay }}
+    </KButton>
+    <template
+      v-if="!hidePopover"
+      #content>
+      <!-- Custom | Relative toggle -->
+      <KSegmentedControl
+        v-if="hasTimePeriods && hasCalendar"
+        v-model="tabName"
+        :options="[
+          { label: 'Relative', value: 'relative' },
+          { label: 'Custom', value: 'custom' }
+        ]"
+        class="w-100 mb-5"
+        data-testid="analytics-time-toggle"
+      />
+      <!-- Single date / time or range readout -->
+      <p
+        v-if="!showCalendar"
+        class="range-display"
+      >{{ fullRangeDisplay }}</p>
```

Comment:
I'm not sure it makes sense to show this when the user is picking a relative timeframe only (e.g. the **Relative time frames only** example) unless you're going to show the actual start and end for what they selected.

![image](https://user-images.githubusercontent.com/2229946/189490031-00f5e930-8dd2-49ce-8bb2-d6c45a97e91b.png)

Reply:
Will fix, it's happening because we didn't pass in `:range="true"`.